ui/src/app/store/application/effects.ts

```diff
@@ -49,8 +50,8 @@ export class ApplicationListEffect {
 
   @Effect()
   loadApplications$ = this.actions.pipe(
-    ofType(loadApplications),
-    switchMap(action =>
+    ofType(loadApplications, loadApplicationsNotLoader),
```

Comment:
what is the fix? loadApplicationsNotLoader and loadApplications are now identical